css/site.styl

```diff
@@ -29,49 +29,108 @@ normalize()
 // GRID
 
 $cols = 12
-$col = 90px
-$gutter = 16px
+$gutter = 18px
 
 .container
   margin: 0 auto
-  width: ($col * $cols)+($gutter * $cols)
+  width: 1150px
 
 .row
-  clearfix()
-
+  box-sizing: border-box
+  display: flex
+  flex: 0 1 auto
+  flex-direction: row
+  flex-wrap: wrap
+  padding-top: $gutter
+  margin-bottom: $gutter
+  margin-right: -($gutter)
```

Comment:
i think better the rows have no margins or paddings. we can set padding-top easily with utility classes.